src/index.js

```diff
@@ -53,11 +52,17 @@ async function start(fields) {
 }
 
 async function authenticate(mail, password) {
+  // Initiate API UserData object, for checkFormLogin token
+  const req1 = await request({
+    url: `https://www.deezer.com/ajax/gw-light.php?method=deezer.getUserData&input=3&api_version=1.0&api_token=&cid=`
+  })
```

Comment:
To avoid the comment, you can use a named function

```js
function async initiateAPIUserDataObjectForCheckFormLoginToken () {
  return request({
    url: `https://www.deezer.com/ajax/gw-light.php?method=deezer.getUserData&input=3&api_version=1.0&api_token=&cid=`
  })
}
```

Reply:
Seems more readeable in one function to me.